editoast/src/models/rolling_stock_model.rs

```diff
@@ -62,10 +77,14 @@ pub struct RollingStockModel {
     #[model(json)]
     pub energy_sources: Vec<EnergySource>,
     pub locked: bool,
-    #[schema(required)]
-    pub electrical_power_startup_time: Option<f64>,
-    #[schema(required)]
-    pub raise_pantograph_time: Option<f64>,
+    //#[schema(required)]
```

Comment:
Removing the `schema(required)` makes the field not required in the OpenAPI. But since we always serialize `None` to `null`, we can ensure the field's always there. Doing so is likely to change a `number | null` to `number | null | undefined` somewhere in the RTK file, maybe raising type errors on the TypeScript side.